src/palace/manager/api/boundless/settings.py

```diff
@@ -7,15 +7,14 @@
     BaseCirculationApiSettings,
     BaseCirculationLoanSettings,
 )
-from palace.manager.integration.configuration.formats import FormatPrioritiesSettings
 from palace.manager.integration.settings import (
     ConfigurationFormItem,
     ConfigurationFormItemType,
     FormField,
 )
 
 
-class BoundlessSettings(BaseCirculationApiSettings, FormatPrioritiesSettings):
+class BoundlessSettings(BaseCirculationApiSettings):
```

Comment:
This was my first attempt at allowing us to set the priority, but it didn't make much sense, since the settings here reference LCP. Removing this, and I'll do a followup PR with a setting specifically for Boundless.